src/herumi/signature.ts

```diff
@@ -17,8 +17,7 @@ export class Signature implements ISignature {
     this.value = value;
   }
 
-  static fromBytes(bytes: Uint8Array, _type?: CoordType, validate = true): Signature {
-    if (validate === false) throw new Error("ValidationSkipNotSupported");
+  static fromBytes(bytes: Uint8Array): Signature {
```

Comment:
Can you keep the signature as previously and repeat the comment here?
```ts
/** 
 * @param type Does not affect `herumi` implementation, always de-serializes to ???
 * @param validate With `herumi` implementation signature validation is always on regardless of this flag.
 */
static fromBytes(bytes: Uint8Array, _type?: CoordType, validate = true): Signature {
```
If you can please figure out what coordinates type herumi de-serializes to and replace it for the `???` above

Reply:
Ok will figure it out 👍

Reply:
as per https://github.com/herumi/mcl/blob/a4fe1155c34cbbd7742cc1313c018f71418d9a81/api.md this is `jacobian` also verified for the sizes that are being allocated, seems in line with `jacobian`